samples/msal-node-samples/ElectronTestApp/package.json

```
@@ -20,10 +20,9 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
-    "@playwright/test": "~1.27.0",
+    "@playwright/test": "~1.28.1",
```

We can make this carat again since the bug appears to be fixed 
```suggestion
    "@playwright/test": "^1.28.1",
```